docs/language/syntax.rst

```diff
@@ -174,13 +178,19 @@ Python-legal names. The steps are as follows:
      same way. If the character doesn't have a name, use ``U`` followed by its
      code point in lowercase hexadecimal.
 
-   Thus, ``green☘`` becomes ``hyx_greenXshamrockX``, ``if`` becomes
-   ``hyx_if``, and ``is_-_has_dashes`` becomes ``hyx_is_XhyphenHminusX_has_dashes``.
+   Thus, ``green☘`` becomes ``hyx_greenXshamrockX`` and
+   ``is_-_has_dashes`` becomes ``hyx_is_XhyphenHminusX_has_dashes``.
```

Comment:
I still think it's worthwhile to say what `if` mangles to. Mangle handles reserved words specially and it should be clear how

Reply:
Not exactly. This is an angels-dancing-on-the-head-of-a-pin issue, but `hy.mangle` ignores the whole reserved-word issue entirely and thus `(hy.mangle "if")` returns `"if"`. The step where we do the Unicode trick occurs in `ast.unparse`.